src/Core/Entity/User.php

```diff
@@ -27,6 +27,7 @@ class User extends StaticEntity
 	protected $updated;
 	protected $role;
 	protected $language;
+	protected $contacts;
```

Comment:
Could you check what happens if someone sends an update with contacts in the payload?
We've seen previous issues where the repo's try to automatically set a user.contacts field in the db and everything breaks.

Reply:
I tested it via the client and it worked but I can add a test for it

Reply:
Ok, I updated one of the tests to ensure that including contacts doesn't cause a problem and changed the user update method to unset contacts explicitly.